src/rpc/masternode.cpp

```diff
@@ -578,6 +579,7 @@ UniValue masternodelist(const JSONRPCRequest& request)
             if (strFilter !="" && strInfo.find(strFilter) == std::string::npos &&
                 strOutpoint.find(strFilter) == std::string::npos) return;
             UniValue objMN(UniValue::VOBJ);
+            objMN.push_back(Pair("protxhash", dmn->proTxHash.ToString()));
```

Comment:
```suggestion
            objMN.push_back(Pair("proTxHash", dmn->proTxHash.ToString()));
```
To keep it consistent with other RPCs.